discovery-infra/tests/test_kube_api.py

```diff
@@ -267,6 +267,12 @@ def download_iso_from_infra_env(infra_env, iso_download_path):
 
 
 def set_single_node_ip(cluster_deployment, nodes, is_ipv4):
+    if len(nodes) != 1:
+        # Conveniently do nothing to allow this function to be
+        # used unconditionally where the number of nodes is not
```

Comment:
Is there a really compelling reason to do this?

I feel like this will just be super confusing when reading the caller.

Reply:
Nah you're right